.github/workflows/build-and-deploy.yml

```diff
@@ -53,6 +53,7 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.CI_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.CI_AWS_SECRET_ACCESS_KEY }}
+          AWS_SESSION_TOKEN: ${{ secrets.CI_AWS_SESSION_TOKEN }}
```

Comment:
This would be used only if using SSO creds. For all other cases, it should be ignored (we'll find out if the PR build fails, I guess!)

Reply:
```suggestion
```

> This would be used only if using SSO creds

Including this without a comment suggests it's required, but it isn't. Since we don't currently have a need for this, I'd suggest leaving it out.

Reply:
Or I'll add a comment that it is optional. :)